source/features/embed-gist-via-iframe.tsx

```diff
@@ -0,0 +1,37 @@
+import select from 'select-dom';
+import features from '../libs/features';
+
+function init(): void {
+	const embedViaScript = select('.select-menu-item')!;
+	const embedViaIframe = embedViaScript.cloneNode(true) as HTMLElement;
+
+	// Modify description to distinguish the two items
+	// This is also consistent with the existing "Clone via HTTPS" and "Clone via SSH" items
+	const viaScript = 'Embed via <script>';
+	select('[data-menu-button]')!.textContent = viaScript;
```

Comment:
Broad selector or this needs to be 
```suggestion
	select('[data-menu-button]', embedViaScript)!.textContent = viaScript;
```

Reply:
That's actually wrong but never mind